pallets/bridge_transfer/src/mock.rs

```diff
@@ -107,10 +104,19 @@ impl bridge::Config for Test {
 	type ProposalLifetime = ProposalLifetime;
 }
 
+parameter_types! {
+	// bridge::derive_resource_id(1, &bridge::hashing::blake2_128(b"PHA"));
+	pub const BridgeTokenId: [u8; 32] = hex!["00000000000000000000000000000063a7e2be78898ba83824b0c0cc8dfb6001"];
```

Comment:
`hex!("aabbcc")` may make more sense. `hex![]` looks like it will accept hex!["aabb", "ccdd"] and generate a `[[u8; _]; _]`